homework4.php

```diff
@@ -0,0 +1,24 @@
+<?php
+//homework4
+function checkPrimeNumber2($n) {
```

Comment:
Please check the hint at https://gitlab.com/pascaliaasia/pav-training/-/blob/basic-training/basic-programing/php/homework-hint/php-hint.md